apidef/migration_test.go

```diff
@@ -589,7 +589,8 @@ func TestAPIDefinition_migrateCustomPluginAuth(t *testing.T) {
 func TestSetDisabledFlags(t *testing.T) {
 	apiDef := APIDefinition{
 		CustomMiddleware: MiddlewareSection{
-			Pre: make([]MiddlewareDefinition, 1),
+			Pre:         make([]MiddlewareDefinition, 1),
+			PostKeyAuth: make([]MiddlewareDefinition, 1),
```

Comment:
When `CustomMiddleware` is empty, the `Pre` value is already `nil`. The `disabled` should not cause a problem in empty `Definition` ?

Reply:
yes, but we're testing `SetDisabledFlags` when `CustomMiddleware` is empty, we're not testing that